tests/pr_test.go

```diff
@@ -21,6 +21,7 @@ var validRegions = []string{
 	"us-east",
 	"ca-mon",
 	"ca-tor",
+	"eu-fr2",
```

Comment:
remove this - we should not be doing automated tests in this region. Manual test is ok